covid19-notebooks/covid19-sir-bayes-model/sir_model.py

```diff
@@ -52,6 +52,7 @@
         "1/22/20":,
     ]
 )[0]
+
 date_data_end = date(year + 2000, month, day)
 date_today = date_data_end + datetime.timedelta(days=1)
 print(
```

Comment:
I strongly suggest setting up a logger instead of print

Reply:
logger added 😊